test/WriterTest.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Hostnet\Component\EntityPlugin;
+
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * @covers Hostnet\Component\EntityPlugin\Writer
+ */
+class WriterTest extends \PHPUnit_Framework_TestCase
```

Comment:
Not a mandatory change, but as of phpunit 5.4 there's a forward compatible namespace